minard/templates/calibdq_tellie_subrun.html

```diff
@@ -0,0 +1,96 @@
+        <div class="panel panel-heading">
+
+            <h3 class="panel-title">Run information for run {{run_number}} subrun {{runInformation["subrun_numbers"][subrun_index]}}</h3>
+        </div>
+        <div class="panel panel-body">
+            <table class="table table-hover"> 
+                <thead>
+                    <tr>
+                        <th>
+                            Parameter
+                        </th>
+                        <th>
+                            Value
+                        </th>
+                    </tr>
+                </thead> 
+                
+                
+                <tbody>
+                <tr>
+                    <td>Expected Number of TELLIE events </td> <td>{{runInformation["expected_tellie_events"][subrun_index]}}</td>
+                </tr>
+                <tr>
+                    <td>Actual Number of TELLIE events </td> <td>{{runInformation["actual_tellie_events"][subrun_index]}}</td>
+                </tr>
+
+                <tr>
+                    <td>Average NHit</td> <td>{{runInformation["average_nhit"][subrun_index]}}</td>
+                </tr>
+                
+                <tr>
+                    <td>Greater Than 1000 Nhit Events </td> <td>{{runInformation["greaterThanMaxNHitEvents"][subrun_index]}}</td>
+                </tr>
+                
+                <tr>
+                    <td>Fibre Firing</td> <td>{{runInformation["fibre_firing"][subrun_index]}}</td>
+                </tr>
+                
+                <tr>
+                    <td>Calculated Fibre Firing</td> <td>{{runInformation["fibre_firing_guess"][subrun_index]}}</td>
+                </tr>
+
+                <tr>
+                    <td>Number of TAC peaks (Should be 3)</td> <td>{{runInformation["peak_number"][subrun_index]}}</td>
+                </tr>
+
+                <tr>
+                    <td>Pre peak TAC count </td> <td>{{runInformation["pre_peak_adc_count"][subrun_index]}}</td>
+                </tr>
+
+                <tr>
+                    <td>Prompt peak TAC count </td> <td>{{runInformation["prompt_peak_adc_count"][subrun_index]}}</td>
+                </tr>
+                
+                <tr>
+                    <td>Late peak TAC count </td> <td>{{runInformation["late_peak_adc_count"][subrun_index]}}</td>
+                </tr>
+                
+                <tr>
+                    <td>Run Time</td> <td>{{runInformation["subrun_run_times"][subrun_index]}}</td>
+                </tr>
+                
+                <tr>
+                    <td>Proportion of TELLIE events with correct pulse delay seperation </td> <td>{{runInformation["pulse_delay_correct_proportion"][subrun_index]}}</td>
+                </tr>
+                </tbody>
+            </table>
+    </div>
+</div>
+<div class="container">
+    <h1>TELLIE Data Quality Plots for run {{run_number}} subrun {{runInformation["subrun_numbers"][subrun_index]}}</h1>
```

Comment:
How are you planning on getting the images onto minard? Is there a reason to find them in the view and then loop over them here instead of just statically linked to a given set of images like is done in https://github.com/snoplus/minard/blob/master/minard/templates/eca_run_detail_PDST.html?